prescience_client/__init__.py

```diff
@@ -36,6 +38,5 @@ def main():
         root_cmd.exec(args)
         exit(0)
 
-    except PrescienceException as exception:
-        exception.print()
```

Comment:
I don't think that it should be remove, otherwise we'll probably have some silent exception.

Reply:
Actually the exception is handled twice, once with the `handle_exception` method in `prescience_config.py` used in the `call` method of `prescience_client`.
One of those should be removed, I chose to remove the one in the main so that the excpetion handling is still done when using the client as a library.